src/js/movies.js

```diff
@@ -1,3 +1,87 @@
+import  movies  from   '../data/movies.json' assert { type: 'json' };
+
+console.log('movies2', movies) 
+
+let moviesList = document.querySelector('.movies-now')
+let futureMovies = document.querySelector('.movies-later ')
+
+
+class Movies {
+    constructor(title,description,country,languages,director, actors, posterUrl,imageUrl,videoUrl,genres,ageRestriction,releaseDate,screeningTimes,tags,rating){
+        this.title = title;
+        this.description = description;
+        this.country=country;
+        this.languages = languages;
+        this.director = director;
+        this.actors = actors;
+        this.posterUrl = posterUrl;
+        this.imageUrl = imageUrl;
+        this.videoUrl = videoUrl;
+        this.genres = genres;
+        this.ageRestriction = ageRestriction;
+        this.releaseDate = releaseDate;
+        this.screeningTimes = screeningTimes;
+        this.tags = tags;
+        this.rating = rating;     
+
+    }
+
+
+     static displayMovies(){
+        movies.movies.forEach(movie =>{
```

Comment:
Another suggestion to make this function a bit more secure. "Movies" comes from outside of the function scope which can lead to problems in the future, if for example the movie data comes from outside the scope of this file (maybe an API object). 
A solution to this is to have "movies" as a parameter on the function "displayMovies" instead and thus making a dependency injection :)